src/bindgen.rs

```diff
@@ -1,16 +1,46 @@
 use emoji;
 use error::Error;
 use progressbar::Step;
-use std::process::Command;
+use std::{path, process::Command};
 use PBAR;
 
-pub fn cargo_install_wasm_bindgen(step: &Step) -> Result<(), Error> {
+static LOCAL_BINDGEN_PATH: &str = "bin/wasm-bindgen";
```

Comment:
You should cfg this so that windows is "bin\\wasm-bindgen". While windows can handle it (sort of? windows pathing is wild technically the yen symbol is a valid path separator) it's best to have an option for it.